contents/blog/the-posthog-array-1-26-0.md

```diff
@@ -0,0 +1,185 @@
+
+![CSV Download](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/csvexport.png)
+
+If you didn't already know, almost all datapoints in PostHog's 'Insights' section are clickable and reveal the users that make up that datapoint.
+
+Well, now those users can be exported in CSV format, so you can use them in other tools or even create a static PostHog cohort from them.
+
+Thanks a lot for building this [c3ho](https://github.com/c3ho)!
+
+### [Request retries for posthog-js](https://github.com/PostHog/posthog-js/issues/199)
+
+Continuing on the libraries theme, a much-requested feature is now live for `posthog-js`: retries!
+
+Requests that fail because of, for example, the client's network connection, or because PostHog is down will now be retried up to 10 times within an hour, making sure you miss as few events as possible! So if your user's internet goes down and comes back up, you'll still receive the events that happened when they were offline.
+
+Also, Neil fixed a bug that had puzzled us and our users for quite a while, preventing requests from being sent to a wrong endpoint (with no impact on tracking). You can [read about how Neil solved this issue on his personal blog](https://neilkakkar.com/debugging-open-source.html).
+
+### New plugins for Redshift, PostgreSQL, Salesforce, and PagerDuty
+
+You heard that right! We've just released 4 new integrations with major platforms to enhance your PostHog experience. 
+
+Export data to Redshift, Postgres, and Salesforce, and leverage the PagerDuty plugin to get alerts when metrics in PostHog cross thresholds you specify!
+
+### New querying experience
+
+![New querying experience](https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/new-querying.png)
+
+We've redesigned and significantly improved the performance of the query builder in PostHog 'Trends'! 
+
+Let us know what you think about it on [Slack](/slack).
+
+### [User Interviews](calendly.com/posthog-feedback)
+
+We’re working hard to improve PostHog and would love to talk to you about your experience with the product.
+
+If you're interested in helping us out, you can schedule a quick 30min call with us [on Calendly](https://calendly.com/posthog-feedback).
+
+Oh, and we're giving away some awesome [PostHog merch](https://merch.posthog.com) as a thank you!
+
+## PostHog News
+
+We onboarded so many people this cycle that it makes sense to make a table for it!
+
+### New joiners
+
+|  Name  |        Role         | 🍍 on 🍕? |                                                                                        Interesting Fact                                                                                        |
+| :----: | :-----------------: | :-----: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
```

Comment:
This is my error, but: the job titles in my interesting fact aren't consistently capitalised. They should probably all be lower case as they aren't the official titles. 

(Since you asked, the official titles were: Entertainment Staff, Waiter, Hotel Services, Salesman, Staff Writer/Senior Staff Writer/Games Editor)